### PR TITLE
Add donate page - DO NOT MERGE YET!

### PR DESCRIPTION
**Pull request by Vaults1**
Please review it.
I have no idea how you edit the background, so if anyone knows how feel free to edit my PR.

**Note to BOD staff:** This PR allows edits from maintainers, please make use of it. Thanks!

### CODE REVIEW

**Review by InspectorGadget — COMMENTED** [review]
Everything looks great btw, i'd suggest using .php extension at the back so it'd look even fancier. I know for the fact that changing it isn't hard. Another suggestion from me is that made a separate dir for each and every page, this could stop being inspect elemented or 'view page source'.

**Review by AvgZing — APPROVED** [review]
Awesome job! I think this should be merged immediately, and the reasons for why to donate are great! I couldn't have said it better myself!

**Review by NiekertDev — APPROVED** [review]
Good work! Merge it as fast as possible!

**Review by Ad5001 — APPROVED** [review]
Great job I like it !

**Comment by AvgZing** [comment]
@RTGDaCoder We can't make it .php because github pages does not support php. We also don't care at all about if they can inspect element because all of the code is open source anyways.

**Comment by AvgZing** [comment]
Just waiting for these four team members!
@Dog2puppy
@Taki21
@HimbeersaftLP 
@TheDragonRing

**Comment by lavender-sys** [comment]
I can never find the review button @github please make it easier to find the review button.

**Review by lavender-sys — APPROVED** [review]

**Comment by HimbeersaftLP** [comment]
> Everything looks great btw, i'd suggest using .php extension at the back so it'd look even fancier. I know for the fact that changing it isn't hard. Another suggestion from me is that made a separate dir for each and every page, this could stop being inspect elemented or 'view page source'.

I don't want to offend you, but did you even notice what crap you just said? lol

**Comment by lavender-sys** [comment]
@HimbeersaftLP I don't understand what he means by people can inspect element.

**Comment by HimbeersaftLP** [comment]
@Dog2puppy Yeah, ppl can always do inspect element, there is no way to hide the source of a html page (flash player doesn't count as html lol)

**Comment by lavender-sys** [comment]
Who uses Flash Player in 2017?

**Comment by HimbeersaftLP** [comment]
@Dog2puppy Some crappy website building tools

**Comment by lavender-sys** [comment]
XD

**Review by HimbeersaftLP — COMMENTED** [review]
I think that loading gif should be replaced with a transparent one (but please take care of copyright when downloading one 😛 )

**Comment by Vaults1** [comment]
@HimbeersaftLP I'm trying to find out how to change the background 😜

**Comment by HimbeersaftLP** [comment]
@Ultie0 Which background?

**Comment by Vaults1** [comment]
@HimbeersaftLP the one behind the text, idk how to make it white.

**Comment by HimbeersaftLP** [comment]
It depends on in which div you put it

**Comment by Thunder33345** [comment]
have to say this, GITHUB IS A STATIC PAGE HOST NOT WEB HOST
but you can use javascript doeeeee

File: index.html [hunk 1]
```diff
@@ -105,7 +106,7 @@ <h4 style="margin-left: 20px;">Staff Members</h4>
           <div class="col-md-4 Ver">
             <i class="fa fa-cog"></i>
             <h4>Highly Customizeable</h4>
-            <p>All of our products are able to be customized through their various settings files. We try to maximize customizeability (is that a word? 😋).</p>
+            <p>All of our products are able to be customized through their various settings files. We try to maximize customizeability (is that a word? ).</p>
```
Comment (HimbeersaftLP) [inline index.html]: Ya broke all da poor emojis 😢

Comment (lavender-sys) [reply index.html]: I guess chrome doesnt like emojis in text areas. I'll fix it.

Comment (lavender-sys) [reply index.html]: And I dont know how to fix it... @Ultie0 @TheRoyalBlock @someone please help me

Comment (AvgZing) [reply index.html]: I think i did it

**Review by AvgZing — DISMISSED** [review]
Please just merge approve this commit. EMOJIS!

**Review by dragonwocky — COMMENTED** [review]
Go ahead and merge. Looks good!

**Review by AvgZing — APPROVED** [review]

**Comment by AvgZing** [comment]
We have a 5 out of 8 vote! Majority rules! I'll let @Ultie0 do the honors.

**Comment by lavender-sys** [comment]
Yay!

**Comment by AvgZing** [comment]
WOOT!

**Pull request merged**